schema/vulntology-json-schema-1.0-draft.json

```diff
@@ -376,7 +378,8 @@
             "type": "object",
             "required": [
                 "hasIdentity",
-                "hasScenario",
+                "hasOriginatingProduct",
```

Comment:
The property `hasOriginatingProduct` defines the product associated with the vulnerability. Without this property, it is not clear which product the vulnerability is exploited against. As a result, this property should be required.